### PR TITLE
Add a test page for WYSIWYG styles.

### PR DESCRIPTION
**Pull request by lewiscollard**
This adds a template for testing every possible WYSIWYG style (it can be viewed through `/frontend/wysiwyg/` when `settings.DEBUG == True`.

Bonus changes:
* Cells of the first row of a table are not treated as a header row - one should use an actual header row if that is required. 
* Default styles are now added for _unclassed_ h[2-6].
* A never-used `wys-Title-xlarge` option is removed from the default settings.
* The `paste` plugin is restored (required for paste as text to work).

### CODE REVIEW

**Review by daniel-clayton — COMMENTED** [review]
image
Does this image work for you?

**Review by daniel-clayton — COMMENTED** [review]
image
I feel like we should just make `hr`s `margin: 1vr 0`

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/frontend/wysiwyg.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{% extends 'frontend/_base.html' %}
+
+{% block main %}
+  <div class="wys-WYSIWYG" style="max-width: 800px">
+    <p class="wys-Standfirst">Standfirst lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```
Comment (daniel-clayton) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/frontend/wysiwyg.html]: Can we include the class `wys-Standfirst` in `wysiwyg.css` even if it just has `color: var(--Colour_Something)`

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py [hunk 2]
```diff
@@ -404,11 +404,6 @@
                     'selector': 'h1,h2,h3,h4,h5,h6',
                     'classes': 'wys-Title-large'
                 },
-                {
-                    'title': 'Extra large',
-                    'selector': 'h1,h2,h3,h4,h5,h6',
-                    'classes': 'wys-Title-extraLarge'
-                }
             ]
```
Comment (daniel-clayton) [inline {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py:407]: Missing the option to select `wys-Standfirst`

**Comment by lewiscollard** [comment]
@daniel-clayton ah of course that image worked on the particular project I copied it from... :/ There's no reasonable way to test that, so I've removed the image from the test (it can't really go wrong anyway).

`<hr>`s in WYSIWYGs are now `margin: 1vr 0`.

**Review by daniel-clayton — CHANGES_REQUESTED** [review]

File: {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py (hunk 2 above)
Comment (lewiscollard) [reply {{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/settings/base.py:407]: My provisional solution to this is to remove the standfirst test in the frontend template, as not all sites use a standfirst style.

**Comment by daniel-clayton** [comment]
Do you want to squash and merge, since the build only failed because of the pytest thing?

**Review by daniel-clayton — APPROVED** [review]

**Review by AidanCurrah — APPROVED** [review]

**Pull request merged**